MIPS/mip-30.md

```diff
@@ -0,0 +1,385 @@
+---
+mip: 30
+title: Product Sunset Approach
+status: Proposed
+author: Dimitri Golecko (@dimsome), Nick Addison (@naddison36), Cesar Chavez (@doncesarts)
+discussions-to:
+created: 2023-02-27
+---
+
+## Simple Summary
+
+This proposal outlines a sunset process and smart contract changes to products as specified in [MIP 29](https://mips.mstable.org/MIPS/mip-29.html).
+
+It is important to note that this proposal is not indented to propose or discuss whether to sunset the products or not but rather in the case that the subsequent proposals to sunset are voted upon there is a set of clear guides and requirements specified to follow.
+
+## Abstract
+
+### Important comment upfront:
+
+- **_This MIP is NOT to discuss whether to sunset or not_**, this will be part of a future proposal. But in light of MIP 29, we need to prepare for the eventuality of the shutdown of the products and how it’s going to be handled on a smart contract level.
+- **_This MIP outlines HOW to sunset the products_** and to offer transparency around the guiding principles, methodology and smart contract changes/upgrades.
+- **_The actual decision to execute this will be done in a future proposal._** A future proposal will be made that is referenced in MIP 29 that will offer options for proposed continuation as well as to pursue the shutdown.
+- In the case that a proposal for Merge/Acquisition/Continuation is successful, this Proposal and the formal proposal resulting from it will be superseded by the winning proposal
+
+This document serves to discuss the guiding principles and methodology for sunsetting the products and smart contracts in accordance with [MIP 29](https://mips.mstable.org/MIPS/mip-29.html).
```

Comment:
"sunsetting the products and smart contracts" or "sunsetting the products and their smart contracts"?